irohad/model/execution/impl/command_executor.cpp

```diff
@@ -71,22 +71,22 @@ bool AddAssetQuantityExecutor::execute(const Command &command,
   auto account_asset = queries.getAccountAsset(add_asset_quantity.account_id,
                                                add_asset_quantity.asset_id);
   if (not account_asset.has_value()) {
-    log_->info("create wallet {} for {}",
-               add_asset_quantity.asset_id,
+    log_->info("create wallet {} for {}", add_asset_quantity.asset_id,
                add_asset_quantity.account_id);
 
     account_asset = AccountAsset();
     account_asset->asset_id = add_asset_quantity.asset_id;
     account_asset->account_id = add_asset_quantity.account_id;
-    account_asset->balance =
-        add_asset_quantity.amount.get_joint_amount(precision);
+    account_asset->balance = add_asset_quantity.amount;
   } else {
-    account_asset = account_asset.value();
-    // TODO: handle non trivial arithmetic
-    auto new_balance = account_asset.value().balance +
-        add_asset_quantity.amount.get_joint_amount(precision);
-    // TODO: handle overflow
-    account_asset->balance = new_balance;
+    auto account_asset_value = account_asset.value();
+    std::cout << account_asset_value.balance.to_string() << " " << add_asset_quantity.amount.to_string() << std::endl;
```

Comment:
Do we need std::cout here? Maybe it is better to use Log

Reply:
It is better to remove it